### ISSUE TITLE
Unable to load 'ActiveCode' and 'Show Code' History from completed chapters

### ISSUE THREAD

**Issue by elexjet** [RunestoneInteractive/RunestoneServer#1226]
I am doing the Foundations of Python Programming and it seems that since the Runestone website was updated recently, I am unable to load my previously written ActiveCode and Show Code history of any chapters I have completed.
I have created an account username on the open Runestone Academy website and I am logged in every time I use the textbook. I am not enrolled in a course in high school or college. 
This is the address when I'm logged in: https://runestone.academy/runestone/static/fopp/index.html

When I created my account, I added 'pythonds' as a course, as I did not know what the course name acronym was for Foundations of Python Programming. Therefore, every time I logged in I was directed to the Problem Solving with Algorithms and Data Structures using Python. I had to open 'Foundations of Python Programming' from the 'Book Links' menu. 
However as at yesterday, as I logged in and tried to access FOPP from the 'Book Links', I received a pop-up message saying something like 'Hey it seems that you have been carried away from your enrolled course, redirecting you to your current course', something like that as I don't really remember. As a result I clicked on 'Change Course' and was able to add FOPP to my list of courses (I was unable to add more than one course when I created my account).
My code was previously saved in my Runestone account as I could access my code on different computers and devices.

Username: aerox
'ActiveCode' and 'Show Code' problem: unable to load past code history using the 'Load History' and 'Show Code' buttons.
Logged in: Yes
Current version of Runestone: 3.4.5 using open version of FOPP
browser: Chrome Version 74.0.3729.108 (Official Build) (64-bit) / Safari iOS 12.2
OS: Windows 8.1 Pro & iOS 12.2

![Progress](https://user-images.githubusercontent.com/15072236/56856473-9eb63000-699e-11e9-8c52-62b7e9b100fb.PNG)
![FOPP load history error](https://user-images.githubusercontent.com/15072236/56856580-1b4a0e00-69a1-11e9-9606-89d3ebfc4404.PNG)
![FOPP show code error](https://user-images.githubusercontent.com/15072236/56856582-256c0c80-69a1-11e9-97e2-0d693ed57035.PNG)

Any help restoring my code history would be greatly appreciated.
Runestone is amazing, thank you so much for creating it!

**Comment by bnmnetp** [comment]
The pop up error message was added a few days ago to prevent just this kind of thing from happening!

Your code was not lost, it was saved with your username and the activecode ID and the course you were registered for — pythonds.  The fact that you had to change courses all the time was a clue that something was not right.  

I updated the database to move your saved code out of pythonds and under fopp, so now that you are registered for fopp and using fopp your code should be there.
